ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/geospatial/bucketising/GeospatialBucketiser.java

```diff
@@ -56,4 +59,25 @@ private List<Coordinate> toCoordinates(Object geoObject) {
 			return List.of();
 		}
 	}
+
+	private List<Object> getFragmentationObjects(Model model, String subjectFilter, String fragmentationPath) {
+		return model
+				.listStatements(null, ResourceFactory.createProperty(fragmentationPath), (Resource) null)
+				.toList()
+				.stream()
+				.filter(statement -> statement.getSubject().toString().matches(subjectFilter))
+				.map(this::getValue)
+				.filter(Optional::isPresent)
+				.map(Optional::get)
+				.toList();
```

Comment:
Same comment as before, line 72 and 36 counter act each other. If it's private and only used here, drop the list conversion